File: Assignment15/program1.c
/*
1.Write a program which accept number from user and return the count of even
digits.
Input   : 2395
Output  : 1
*/

#include<stdio.h>

int CountEven(int iValue)
{
    int iDigit = 0;
    int iCount = 0;

    while(iValue != 0)
    {
        iDigit = iValue % 10;
        iValue = iValue / 10;

        if(iDigit % 2 == 0)
        {
            iCount = iCount + 1;
        }
    }
    return iCount;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter the value : ");
    scanf("%d",&iValue);

    iRet = CountEven(iValue);
    printf("The count of Even Number is : %d\n",iRet);

    return 0;
}

File: Assignment21/program1.java

// 1. Write a program to calculate the product of digits of a number.

package Assignment21;

import java.util.Scanner;

public class program1 
{
    public int Product_Digit(int iValue)
    {
        int iDigit = 0,iprod = 1;

        while (iValue != 0)
        {
            iDigit = iValue % 10;
            iprod = iprod * iDigit;
            iValue = iValue / 10;
        }
        return iprod;
    }

    public static void  main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        program1 pobj = new program1();

        int iValue = 0,iRet = 0;

        System.out.println("Enter the Number : ");
        iValue = sobj.nextInt();

        iRet = pobj.Product_Digit(iValue);
        System.out.println("Product of all Digits in the Number is : "+iRet);

        sobj.close();
    }
}
